jenkins/databricks/run-tests.py

```diff
@@ -64,6 +65,8 @@ def main():
           base_spark_pom_version = arg
       elif opt in ('-b', '--bulidprofiles'):
           build_profiles = arg
+      elif opt in ('-u', '--stagingrepo'):
+          staging_repo = arg
```

Comment:
In Databricks nightly snapshot build, staging_repo=''
In Databricks release build, staging_repo='Sonatype'

Reply:
staging_repo arg here need an argument?  doesn't look like its used other then for a boolean which cuases us to set the build_arg. If that is the case perhaps use "u" instead of "u:" and set staging_repo here to something like "USE_STAGING" or True for the build.sh script to pick up and act on.